docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc

```diff
@@ -51,6 +51,54 @@ In addition to the https://download.eclipse.org/microprofile/microprofile-open-a
 |true
 |Indicates whether auto-generated Server records are absolute or relative to the location of the OpenAPI endpoint.  If absolute, WildFly will generate Server records including the protocols, hosts, and ports at which the given deployment is accessible.
 
+|mp.openapi.extensions._server-name_._host-name_.externalDocs.url
+|_None_
+|Defines the external documentation URL of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.externalDocs.description
+|_None_
+|Defines the external documentation description of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.contact.name
+|_None_
+|Defines the contact name of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.contact.email
+|_None_
+|Defines the contact email of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.license.name
+|_None_
+|Defines the license name of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.license.identifier
+|_None_
+|Defines the license identifier of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.license.url
+|_None_
+|Defines the license URL of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.title
+|_None_
+|Defines the title of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.description
+|_None_
+|Defines the description of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.summary
+|_None_
+|Defines the summary of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.termsOfService
+|_None_
+|Defines the terms of service of the OpenAPI document provided by this host.
+
+|mp.openapi.extensions._server-name_._host-name_.info.openapi
+|{OpenAPIVersion}.0
+|Defines the version of the OpenAPI document provided by this host.
```

Comment:
Do all these apply in the non-discordant case? If not they may all need a few words and a link to the NOTE text.

Reply:
Yes.